docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  rabbit:
+    build: 
+      dockerfile: ./docker/service_rabbitmq/Dockerfile.rabbitmq
+      context: ./
+    environment:
+      - RABBITMQ_DEFAULT_USER=jp
+      - RABBITMQ_DEFAULT_PASS=admin
```

Comment:
Elsewhere it looks for these without "DEFAULT" - so changing to:

```yaml
    environment:
      - RABBITMQ_USER=jp
      - RABBITMQ_PASS=admin
 ```

seemed to resolve of of the errors